pkg/proxy/handlers/spacelister_test.go

```diff
@@ -0,0 +1,102 @@
+package handlers_test
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/codeready-toolchain/registration-service/pkg/application/service"
+	"github.com/codeready-toolchain/registration-service/pkg/context"
+	"github.com/codeready-toolchain/registration-service/pkg/proxy/handlers"
+	"github.com/codeready-toolchain/registration-service/pkg/signup"
+	"github.com/codeready-toolchain/registration-service/test/fake"
+	"k8s.io/apimachinery/pkg/labels"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/labstack/echo/v4"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Work in progress
```

Comment:
These tests do look like WIP :)
Let's add more tests to cover:
- cases when there are multiple space bindings (shared spaces) including some noise (spaces and space bindings belonging to other users)
- error handling